tests/test_json_schema.py

```diff
@@ -2357,7 +2368,7 @@ class LocationBase(BaseModel):
     }
 
 
-def test_advanced_generic_schema():
+def test_advanced_generic_schema():  # noqa: C901
```

Comment:
yeah this test is a beast

Reply:
I think this came from Sebastian originally lol